themes/default/content/blog/pulumi-challenge-september/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Pulumi Challenge: One Quickstart to Rule them All"
+
+date: 2022-09-30T9:24:50-07:00
+
+meta_desc: Try Pulumi out for yourself by taking a Pulumi Challenge! Learn how it works, and get super secret swag in return.
+
+meta_image: meta.png
+
+authors:
+    - kat-cosgrove
+
+tags:
+    - pulumi-challenge
+    - announcements
+
+---
+
+The Pulumi Challenge continues! We think Pulumi is an amazing IaC tool that increases developer velocity and handles the scale of the cloud with ease. To prove it, take this month's challenge and win another unique piece of swag!
+
+<!--more-->
+
+![Pulumi Challenge banner image](challenge_banner.png)
+
+The [Pulumi Challenge]({{< relref "/challenge" >}}) is a series of specialized tutorials, released monthly. Each challenge will guide you through building a real-world application, some of them useful and some of them silly, showcasing the varied ways Pulumi can be a lifesaver for your team at work or your side-projects at home. At the end, you'll have something you can use, and we'll send you swag unique to each challenge too!
+
+## How it Works
+
+Work through the Challenge tutorial to build a real-world application, personalizing it as you go. You'll learn how Pulumi works to ease integration and infrastructure deployment through providers for major cloud providers, Kubernetes, observability tools, CI/CD systems, dynamic providers, reusable component resources, and more.
+
+Once a Challenge is complete, our swag provider will kick off and send you the unique item for that Challenge. Every challenge offers something different, so complete them all as we release them!
+
+There's another way to get more swag, though: Write a 500-word blog article or post a 3-minute video about your Pulumi Challenge project, and we'll send you something even nicer.
+
+## September Challenge
+
+Our second Pulumi Challenge is [One Quickstart to Rule Them All]({{< relref "/challenge/one-quickstart" >}}). We'll walk you through building a serverless application with AWS API Gateway and Lambda. Thanks to the use of Pulumi Architecture Templates, you don't even have to write the code yourself! You'll also learn about Policy Packs with Pulumi, allowing you to gate deployments with security or business logic policies. As a reward for completion, we'll send you a limited edition prize!
```

Comment:
Worth linking to the arch templates homepage (https://www.pulumi.com/templates/) here?